### PR TITLE
CORE-1285: accept incoming timestamps in ISO8601 date-time format eit…

### PR DESCRIPTION
**Pull request by slr71**
…her with or without milliseconds

### CODE REVIEW

File: test/mescal/util_test.clj [hunk 1]
```diff
@@ -10,3 +10,17 @@
     (is (= "foo%20quux/bar%20baz" (encode-path "/iplant/home" "/iplant/home/foo quux/bar baz"))))
   (testing "Encoding other characters"
     (is (= "foo/bar%3Abaz" (encode-path "/iplant/home" "/iplant/home/foo/bar:baz")))))
+
+(deftest timestamp-test
+  (testing "to-utc for a timestamp without milliseconds"
+    (is (= "2021-02-23T21:50:33+00:00" (to-utc "2021-02-23T21:50:33Z")))
+    (is (= "2021-02-23T22:50:33+00:00" (to-utc "2021-02-23T21:50:33-01:00"))))
+  (testing "to-utic for a timestamp with milliseconds"
```
Comment (ianmcorvidae) [inline test/mescal/util_test.clj:18]: tpyo in the test description here

**Review by ianmcorvidae — APPROVED** [review]
Looks good to me. Found a single-character fix that doesn't affect functionality :laughing:

File: test/mescal/util_test.clj (hunk 1 above)
Comment (slr71) [reply test/mescal/util_test.clj:18]: Oosp! Thanks for catching taht! 😄

**Review by psarando — APPROVED** [review]
LGTM 👍

**Comment by slr71** [comment]
Thanks for the reviews!

**Pull request merged**